release.md

```diff
@@ -8,7 +8,7 @@ Before starting the release process, verify the following:
 - [All Github Actions Tests are green on main](https://github.com/alteryx/featuretools/actions?query=branch%3Amain).
 - EvalML Tests are green with Featuretools main
   - [![Unit Tests - EvalML with Featuretools main branch](https://github.com/alteryx/evalml/actions/workflows/unit_tests_with_featuretools_main_branch.yaml/badge.svg?branch=main)](https://github.com/alteryx/evalml/actions/workflows/unit_tests_with_featuretools_main_branch.yaml)
-- The latest Looking Glass performance tests runs should not show any significant performance regressions. If performance regressions have been identified, review the results with the development team to decide on the proper course of action before proceeding with the release.
+- Looking Glass performance tests runs should not show any significant performance regressions when comparing the last commit on `main` with the previous release of Featuretools. See below for instructions on manually launching the performance tests runs. If performance regressions have been identified, review the results with the development team to decide on the proper course of action before proceeding with the release.
```

Comment:
Let's be more explicit here. The person releasing has to kickoff the performance tests, and look at the results.

Reply:
I worded it this way because this is in a list of things to verify, not a list of actions for the person releasing to do. Do you think the action required will be clear enough if we move the detailed instructions to Step 1 in the release process?